website/www/site/content/en/documentation/basics.md

```diff
@@ -42,6 +42,10 @@ understand an important set of core concepts:
    them to a runner.
  * [_Runner_](#runner) - A runner runs a Beam pipeline using the capabilities of
    your chosen data processing engine.
+ * [_Trigger_](#trigger) - A trigger determines when to aggregate the results of
+   each window.
+ * [_State and timers_](#state-and-timers) - Per-key state and timer callbacks
```

Comment:
Per-key state and timer callbacks are the lower level primitives that give you full control of..